API/GetWordLinkRecording.php

```diff
@@ -0,0 +1,49 @@
+<?php
+require_once('utils/Model.php');
+require_once('utils/Respond.php');
+use storyproducer\Respond;
+session_start();
+
+
+if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+    if (!(array_key_exists('PhoneId', $_POST) &&
+        array_key_exists('term', $_POST))) {
+        RespondWithError(400, 'This endpoint requires PhoneId and term');
+    }
+
+    $androidId = htmlspecialchars($_POST['PhoneId']);
+    $term = htmlspecialchars($_POST['term']);
+
+    $model = new Model();
+
+    # check if project exists for the given androidId
+    $projectId = $model->GetProjectId($androidId);
+    if (!$projectId) {
+        RespondWithError(400, "Please register a project using /API/RegisterPhone.php before using /API/UploadWordLinkRecording.php");
```

Comment:
Can you return a 404 here with the message: "The requested term has not been registered.". Here, we don't want to ask the user to register because the app should register, not the web user.